pallets/pool-fees/src/lib.rs

```diff
@@ -407,6 +410,7 @@ pub mod pallet {
 			})?;
 
 			Self::deposit_event(Event::<T>::Charged {
+				pool_id: Self::get_pool_id(&fee_id)?,
```

Comment:
Yeah I thought about returning this as part of `mutate_active_fee` but I thought that the value should be cached anyway making this storage read free. Is that not the case?

Reply:
The weight will not assume it to be cached IIRC. But not sure. Anyays, fine to leave it like that.

Reply:
Good point, that would not be cached AFAIK. Refactored in [409ae02](https://github.com/centrifuge/centrifuge-chain/pull/1718/commits/409ae02bbbf6263d9cb2d22a5aeed07411c517f9).

Mainly went this path because in FI v2 we do a lot of same storage reads for one operation. Looking forward to the benchmarking results :D